shacl12-profiling/index.html

```diff
@@ -438,6 +438,50 @@
 
 		<section id="introduction">
 			<h2>Introduction</h2>
+			<section id="what-is">
+				<h3>What is profiling?</h3>
+				<p>
+					Profiling is the act of creating a "profile" of something.
+				</p>
+				<p>
+					Generically, in English, a "profile" of something is:
+				</p>
+				<p style="padding: 0 50px 0 50px;">
+					The outline of a physical object or feature, or a representation of this
+					<br /><br />- Oxford English dictionary, use of the word "profile" since the 17th century
+				</p>
+				<p>
+					Within the world of data, a derived definition of "profile" consistent with the above is:
+				</p>
+				<p>
+					A summary or an extraction
+				</p>
+				<p>
+					In this definition, the essence of the English word is retained since a summary or extraction of/from a data object <em>may</em> be an outline of it, for example a 2D representation of a 3D spatial object or a statistical summary of a dataset with lots of parts.
```

Comment:
```suggestion
					In this definition, the essence of the English word is retained, since a summary or extraction of or from a data object <em>may</em> be an outline of it; for example, a 2D representation of a 3D spatial object. or a statistical summary of a dataset having lots of parts.
```